src/current/v24.3/sso-sql.md

```diff
@@ -56,122 +56,93 @@ Cluster Setting | Description
 
-    The ID of your cluster as specified by the IdP, or a JSON array of such names. This must match `server.oidc_authentication.client_id`; refer to [Single Sign-on (SSO) for DB Console]({% link {{ page.version.version }}/sso-db-console.md %}).
+    Many third-party token issuers, including GCP and Azure, default to creating tokens with a generic default audience. We recommend limiting the scope of access tokens if possible.
 
-    {{site.data.alerts.callout_danger}}
-    Many third-party token issuers, including GCP and Azure, will by default create tokens with a generic default audience. It is best practice to limit the scope of access tokens as much as possible, so if possible, we recommend issuing tokens with only the required audience value corresponding to the `audience` configured on the cluster.
-
-    By extension, if your provider allows you to specify scopes or permissions on the token, you should specify these as restrictively as possible, while still allowing for the functions intended for the service account or user.
-    {{site.data.alerts.end}}
+    By extension, if your provider allows you to specify scopes or permissions on the token, we recommend configuring the scopes or permissions to be as restrictive as possible.
 
     {% include_cached copy-clipboard.html %}
     ~~~sql
     SET CLUSTER SETTING server.jwt_authentication.audience = '984901724939-njig7lkv7k724rbv2hllvr4of8ul7th7.apps.googleusercontent.com';
     ~~~
 
-1. `server.jwt_authentication.claim`
-    This setting determines which field in the JWT your cluster will use to the identity to a SQL user. The value of `email` indicates the email field in the JWT.
+1. Configure the field in the JWT that contains the email address that corresponds to a SQL user.
 
     {% include_cached copy-clipboard.html %}
     ~~~sql
     SET CLUSTER SETTING server.jwt_authentication.claim = 'email';
     ~~~
 
-1. `server.jwt_authentication.jwks`
-
-    Add your IdP's public signing key to your cluster's list of accepted signing JSON web keys (JWKS), under the `jwks` setting. This is a [JWK](https://www.rfc-editor.org/rfc/rfc7517) formatted single key or key set, containing the public keys for SSO token issuers/IdPs that will be accepted by your cluster. This list must include a given IdP, or the cluster will reject JWTs issued by it. IdPs serve their public certificates and other required information at `https://{ domain }/.well-known/openid-configuration`.
+1. Add your IdP's public signing key, which you found previously, to your cluster's list of acceptable signing JSON web keys (JWKS). Your cluster will reject JWTs that are not signed by a configured key. Your cluster will reject JWTs that are not signed by a signing key that is listed in `server.jwt_authentication.jwks`. If you want to [enable automatic fetching of signing keys](#configuration-examples) instead of configuring static signing keys, skip this step.
```

Comment:
Was it your intention have the following sentences start in the same way:

> "Your cluster will reject JWTs that are not signed by a configured key. Your cluster will reject JWTs that are not signed by a signing key that is listed in `server.jwt_authentication.jwks`."

Reply:
Nope, removed the first sentence.